doc/src/tutorial/001_adding_lalrpop.md

```diff
@@ -61,11 +61,19 @@ fn main() {
 ```
 
 The function `process_root` processes your `src` directory, converting
-all `lalrpop` files into `rs` files. It is smart enough to check
-timestamps and do nothing if the `rs` file is newer than the `lalrpop`
-file, and to mark the generated `rs` file as read-only. It returns an
+all `lalrpop` files into `rs` files, and saving them to `OUT_DIR`. It is smart
+enough to check timestamps and do nothing if the `rs` file is newer than the
+`lalrpop` file, and to mark the generated `rs` file as read-only. It returns an
 `io::Result<()>`, so the `unwrap()` call just asserts that no
 file-system errors occurred.
 
-*NOTE:* On Windows, the necessary APIs are not yet stable, so
+You can include the generated code somewhere in your Rust code. For example, if
+the source grammar were to be called `grammar.lalrpop`, you could include the
+generated `grammar.rs` file like so:
+
+```rust
+include!(concat!(env!("OUT_DIR"), "/grammar.rs"));
```

Comment:
What is the purpose of including the generated gammar like this? The wording sounds like this is the normal way of using the generated grammar, while one should probably prefer `lalrpop_mod!` instead.

Reply:
Could also link to the https://lalrpop.github.io/lalrpop/generate_in_source.html section as a way to get similar behavior I think.

Reply:
This is the very first time I've hearing of `lalrpop_mod!`. The documentation search shows it appears in a few code samples without explicitly mentioning what it does: https://lalrpop.github.io/lalrpop/index.html?search=Lalrpop_mod

(Only one example references it in text, saying 'it's not useful' when generating to the source directory.)

Reply:
(So yeah, this should probably appear more often in the docs. I'll update my PR when I have the chance.)

Reply:
Oh, I thought `lalrpop_mod` was part of the documentation in a clear way...thanks @galexite, mentioning `lalrpop_mod` is a most wanted addition then. I just assumed `lalrpop_mod` was known and that you wanted to achieve something different, but I couldn't really figure out what.